wpiunits/src/main/java/edu/wpi/first/units/Measure.java

```diff
@@ -124,6 +124,19 @@ default U baseUnit() {
    */
   Measure<U> unaryMinus();
 
+  /**
+   * Returns a measure equivalent to this one equal to zero minus its current value. For non-linear
+   * unit types like temperature, the zero point is treated as the zero value of the base unit (eg
+   * Kelvin). In effect, this means code like {@code Celsius.of(10).negate()} returns a value
+   * equivalent to -10 Kelvin, and <i>not</i> -10° Celsius.
+   *
+   * @return a measure equal to zero minus this measure
+   * @deprecated use unaryMinus() instead. This was renamed for consistancy with other WPILib
+   *     classes like Rotation2d
+   */
+  @SuppressWarnings("dep-ann")
```

Comment:
It was because the javadoc had the deprecated, but there wasn't a deprecation annotation which was done at the implementing side to avoid issues with deprecations warnings. But it's OBE with the default implementation.